src/main/scala/org/dia/algorithms/mcc/MCCEdge.scala

```diff
@@ -19,21 +19,29 @@ package org.dia.algorithms.mcc
 
 class MCCEdge(var srcNode : MCCNode, var destNode: MCCNode, var weight : Double) extends Serializable {
 
+  var areaOverlap: Double = 0.0
+
```

Comment:
@sujen1412 var areaOverlap is a counter correct? If so, it would be better to declare as an Int.